server/src/controllers/configController.js

```diff
@@ -1,11 +1,24 @@
 import Config from '../models/config.js';
 
 export function createOne(request, response) {
-  const newConfig = new Config({});
+  const newConfig = new Config({
+    "data.serverType": request.body.data.serverType,
+    "data.appName": request.body.data.appName,
+    "data.serverSettings.port": request.body.data.serverSettings.port,
+    "data.routers": [{
```

Comment:
You probably already know this, but this will only pass in a single router. Especially due to things like:  `request.body.data.routers[0].id,`, which obviously just selects the first router.